src/package_manager/apt.ts

```diff
@@ -104,11 +113,12 @@ async function determineDistribCodename(): Promise<string> {
  *
  * @returns Promise<number> exit code
  */
-export async function installAptDependencies(): Promise<number> {
-	let aptPackages: string[] = aptDependencies;
+export async function installAptDependencies(includeConnext = false): Promise<number> {
+	let aptPackages: string[] = includeConnext ?
+                aptDependenciesNC.concat(aptDependencies) : aptDependencies;
```

Comment:
`aptDependencies.concat([CONNEXT_APT_PACKAGE_NAME])` would probably be more appropriate. The `NC` vs `connext` naming is going to be prone to rot.